apps/ledger-live-mobile/android/app/src/main/java/com/ledger/live/MainActivity.java

```diff
@@ -102,6 +102,9 @@ public void onPrimaryClipChanged() {
             });
         }
 
+        I18nUtil sharedI18nUtilInstance = I18nUtil.getInstance();
```

Comment:
Shouldn't you import `import com.facebook.react.modules.i18nmanager.I18nUtil;
` ?